target/linux/ipq40xx/files-6.12/arch/arm/boot/dts/qcom/qcom-ipq4029-z3.dts

```diff
@@ -0,0 +1,114 @@
+// SPDX-License-Identifier: GPL-2.0-only
+// Device Tree Source for Meraki Z3 (Fuzzy Cricket)
+
+#include "qcom-ipq4029-wired-qca-common.dtsi"
+
+/ {
+	model = "Meraki Z3 Router";
+	compatible = "meraki,z3";
+	leds {
+		compatible = "gpio-leds";
+
+		power_orange: power {
+			color = <LED_COLOR_ID_AMBER>;
+			function = LED_FUNCTION_POWER;
+			label = "orange:power";
+			gpios = <&tlmm 49 GPIO_ACTIVE_LOW>;
+			panic-indicator;
+		};
+
+		led-1 {
+			/* WAN left */
+			color = <LED_COLOR_ID_GREEN>;
+			function = LED_FUNCTION_WAN;
+			function-enumerator = <0>;
+			gpios = <&tlmm 46 GPIO_ACTIVE_LOW>;
+		};
+
+		led-2 {
+			/* WAN right */
+			color = <LED_COLOR_ID_GREEN>;
+			function = LED_FUNCTION_WAN;
+			function-enumerator = <1>;
+			gpios = <&tlmm 30 GPIO_ACTIVE_LOW>;
+		};
+
+		led-3 {
+			/* port 1 left */
+			color = <LED_COLOR_ID_GREEN>;
+			function = LED_FUNCTION_LAN;
+			function-enumerator = <10>;
```

Comment:
Sorry, but what is the logic behind making this 10?

Same goes for others as to me it makes no sense for it to not just be 2 etc